storio/src/main/java/com/pushtorefresh/storio/Loggi.java

```diff
@@ -0,0 +1,276 @@
+package com.pushtorefresh.storio;
+
+import android.support.annotation.NonNull;
+import android.text.TextUtils;
+import android.util.Log;
+
+/**
+ * Simple android Log proxy, includes default TAG and provides ability to turn logs on/off
+ * <p/>
+ * Call it "Loggi, come here!", you can think that Loggi is your little dog :)
+ */
+@SuppressWarnings({ "PMD.ProtectLogD", "PMD.ProtectLogV" })
+public class Loggi {
```

Comment:
There is no any `author` tags in storIO's javadocs, but it's `artem_zin`'s creation. 
Have you ever heard about him?